### PR TITLE
RELAY page: some tidy

### PR DESCRIPTION
**Pull request by olliw42**
title says it

### CODE REVIEW

File: docs/RELAY.md [hunk 1]
```diff
@@ -4,20 +4,19 @@
...
 **Important: Relay functionality is experimental and currently only available in the branch 'dev-relay', pre-compiled binaries can be found in the firmware folder ([here](https://github.com/olliw42/mLRS/tree/dev-relay/firmware)).**
 
-
 mLRS allows for a relay setup in which a main Tx, Rx pair is used with a secondary Tx, Rx pair to offer flexibility when locating the main Tx module. The relay setup will forward the RC channels from the radio to the flight controller while also sending the link statistics and CRSF sensors of the main Tx, Rx pair back to the radio. A relay setup is beneficial by allowing you to place the main Tx module on your antenna tracker while still being able to control the vehicle from the radio.
 
 For a relay setup is recommended to:
-- Use a lower frequency which is better suited for long range for the main Tx, Rx pair
-- Use different frequencies for the main and secondary Tx, Rx pairs to avoid interference
-    - If using the same frequency, then different bind phrases need to be used for the pairs
+- Use a lower RF frequency which is better suited for long range for the main Tx, Rx pair
```
Comment (jlpoltrack) [inline docs/RELAY.md:10]: To try and use 'RF band' more consistently, perhaps

'Use a RF band with a lower frequency which will be better suited for long range for the main Tx, Rx pair'

**Comment by jlpoltrack** [comment]
Looks good, just one piece of feedback.

File: docs/RELAY.md (hunk 1 above)
Comment (olliw42) [reply docs/RELAY.md]: GREAT. done.

**Comment by jlpoltrack** [comment]
Thanks - merging.

**Pull request merged**